renderdoc/driver/d3d12/d3d12_command_list_wrap.cpp

```diff
@@ -3394,10 +3388,8 @@ bool WrappedID3D12GraphicsCommandList::Serialise_ExecuteBundle(SerialiserType &s
       {
         ID3D12GraphicsCommandListX *list = m_Cmd->RerecordCmdList(m_Cmd->m_LastCmdListID);
 
-        uint32_t eventId = m_Cmd->HandlePreCallback(list, true);
-
+        uint32_t eventId = m_Cmd->HandlePreCallback(list, ActionFlags::Drawcall);
```

Comment:
I don't think bundles should be categorised as drawcalls, this should probably be `Misc` with `ActionFlags::CmdList`.

Reply:
I also feel weird when modifying this, because the parameter in the main branch code is set to true, which means this is a dispatch action. Maybe we need to add new enumeration values, such as the ActionFlags::CmdList you mentioned.